webpack.config.js

```diff
@@ -0,0 +1,50 @@
+const path = require('path');
```

Comment:
I don't know anything about webpack configuration--can I trust you that this is all perfect & wonderful?

Reply:
Webpack looks good.